src/main/java/devmountain/group2/entities/UserEntity.java

```diff
@@ -24,7 +24,7 @@ public class UserEntity {
     @Column(unique = true)
     private String password;
 
-// need to implement one to many with availability table
+// TODO do we need to implement one to many with reservation entity?
```

Comment:
@ehavener Just want to confirm that a user can make multiple reservations correct?

Reply:
Yes that is how it is currently implemented